samples/apps/spring-petclinic/Dockerfile.app

```diff
@@ -2,7 +2,7 @@
 FROM --platform=amd64 maven:3.8.4-openjdk-11 as builder
 
 ARG PETCLINIC_REPO=https://github.com/spring-projects/spring-petclinic.git
-ARG PETCLINIC_COMMIT=a7439c74ea718c4f59fe6c7c643c4afe59d7e718
+ARG PETCLINIC_COMMIT=9cb8dde9b8b70218cb666447cbf1342bce7a473c
```

Comment:
why this change - did we build and publish the version of the app?

Reply:
this shouldn't be part of this PR, I missed that, thx - will remove, thx